### ISSUE TITLE
USB access on stm32f105 series

### ISSUE THREAD

**Issue by BBScholar** [stm32-rs/stm32f1xx-hal#310]
Is there a reason the usb module is disabled for stm32f105 series MCUs? The datasheet says it supports USB OTG FS.
```
#[cfg(all(
    feature = "stm32-usbd",
    any(feature = "stm32f102", feature = "stm32f103")
))]
pub mod usb;
```
I'm relatively new to embedded, so forgive me if this is a dumb question.

**Comment by burrbull** [comment]
I think the main reason is connectivity devices are not tested enough.
Also [stm32-usbd](https://github.com/stm32-rs/stm32-usbd) uses its own register map (`pac`) so the relevant place for this issue is there.

**Comment by TeXitoi** [comment]
I think that the silicon for the usb part of 105 is not the same as the silicon for the others, as 105 support OTG, but not the others. Thus, that's a different driver that is needed. The needed driver might be https://crates.io/crates/synopsys-usb-otg

@Disasm what do you think?

**Comment by Disasm** [comment]
That's true, `synopsys-usb-otg` should work with this chip, but I haven't tried this pair yet.

**Comment by WagnerPMC** [comment]
> That's true, `synopsys-usb-otg` should work with this chip, but I haven't tried this pair yet.

Hello,
Have you tried to pair synopsis-usb-otg with F105/F107?
